Pcap++/header/PcapFilter.h

```diff
@@ -243,19 +243,26 @@ namespace pcpp
 	class IPFilter : public IFilterWithDirection
 	{
 	private:
-		std::string m_Address;
+		static constexpr int NO_Len_VALUE = 0;
```

Comment:
the way of mixing uppercase and lowercase looks weird. please check how other code wrote.

Reply:
Fixed 00e7408